local-volume/bootstrapper/deployment/kubernetes/bootstrapper.yaml

```diff
@@ -1,21 +1,21 @@
 apiVersion: v1
 kind: Pod
 metadata:
-  name: local-volume-bootstrapper
+  name: local-volume-provisioner-bootstrap
   namespace: kube-system
   labels:
-    app: local-volume-bootstrapper
+    app: local-volume-provisioner-bootstrap
 spec:
   restartPolicy: Never
   serviceAccountName: local-storage-bootstrapper
   containers:
   - name: bootstrapper
-    image: local-volume-bootstrapper:dev
+    image: quay.io/external_storage/local-volume-provisioner-bootstrap
```

Comment:
updated to latest.

What about versioning. For simplicity, we can use same version number to bootstrap as well?, i.e.
```
quay.io/external_storage/local-volume-provisioner-bootstrap:1.0.0
```